ac1/_sources/regression_and_line_of_best_fit/making_predictions_with_the_regression_line.rst

```diff
@@ -65,11 +65,3 @@ that range is extrapolation. Because there is only one school with an average
 SAT score that low, using this regression line is not going to give a reliable
 (or even reasonable) prediction. Finding regression lines and making predictions
 can be difficult with such outliers.
-
-What if, instead, you try to predict completion rate based on the percentage of
-students receiving federal loans? As the correlation between the percentage of
-students receiving federal loans and completion rate is much smaller than the
-correlation between average SAT score and completion rate, any predictions made
-using the correlation between these variables will be less accurate. This is
-reflected in the scatter plot as well, where the data points are much more
-spread out.
```

Comment:
@raskutti This part of the textbook did not seem to make sense because it refers to SAT score and completion rate much later (perhaps because the sections were re-ordered?) than the scatterplots were introduced in the text. Originally in the module b draft, it came much earlier than the scatterplot was introduced, so it was a bit confusing flow-wise. I'm not sure this paragraph adds that much to the content, so  would it be fine to just delete it?